generators/app/tasks/appSetup/multipleEnvIos/createSchemes.js

```diff
@@ -0,0 +1,12 @@
+const schemeBase = require('./schemeBase');
+
+module.exports = function createSchemes() {
+  const commonPath = schemeName =>
+    `${this.projectName}/ios/${this.projectName}.xcodeproj/xcshareddata/xcschemes/${schemeName}.xcscheme`;
+  this.fs.write(commonPath('qa'), schemeBase.bind(this)('dev'));
```

Comment:
is this line ok? or should it be `schemeBase.bind(this)('qa')`?

Reply:
They are different schemes but with the same .env file

Reply:
We'd have "qa" scheme and "dev" scheme and both would use the `.env.dev` file